src/Components/LikedVideoCard/Likevideocard.css

```diff
@@ -0,0 +1,98 @@
+.liked-video-card-container {
+  height: 15rem;
+  width: 40rem;
+  margin: 2rem;
+  box-shadow: rgba(82, 80, 80, 0.24) 0px 3px 8px;
+  cursor: pointer;
+  position: relative;
+  border: 1px solid var(--dark-stone);
+}
+
+.thumbanail-image {
+  height: 12rem;
+  width: 16.8rem;
+  object-fit: cover;
+  margin: 1rem;
+}
+.liked-video-duration {
+  background-color: rgb(82, 81, 81);
+  color: white;
+  font-size: 1.2rem;
+  padding: 2px;
+  width: 26px;
+  position: absolute;
+  left: 14.4rem;
+  top: 11rem;
+}
+.liked-video-description-container {
+  width: inherit;
+  gap: 10px;
+}
+.liked-video-title {
+  font-size: 1.6rem;
+  margin: -4rem 0 0 -1rem;
+  font-weight: 500;
+  overflow: hidden;
+  text-align: left;
+  text-overflow: ellipsis;
+  display: -webkit-box;
+  -webkit-line-clamp: 2;
+  max-width: 20rem;
+  -webkit-box-orient: vertical;
+}
+.liked-video-channelname {
+  font-size: 1.4rem;
+  align-self: flex-start;
+  margin: 1rem 0;
+}
+.liked-video-icons-container {
+  margin: 0.3rem 1rem -4rem 6rem;
+  width: 10rem;
+  justify-content: space-between;
+}
+.liked-video-icons {
+  font-size: 2rem;
+  justify-self: flex-end;
+  cursor: pointer;
+}
+@media only screen and (min-width: 320px) and (max-width: 880px) and (-webkit-min-device-pixel-ratio: 2) {
```

Comment:
liked you are using `min-width` and `max-width` to define proper range for media queries.